module/renderer/src/commonMain/kotlin/org/cru/godtools/shared/renderer/content/RenderTabs.kt

```diff
@@ -11,22 +14,30 @@ import androidx.compose.runtime.Composable
 import androidx.compose.runtime.mutableStateOf
 import androidx.compose.runtime.remember
 import androidx.compose.ui.Modifier
+import androidx.compose.ui.draw.clip
 import androidx.compose.ui.graphics.Color
+import androidx.compose.ui.unit.dp
+import com.github.ajalt.colormath.extensions.android.composecolor.toComposeColor
 import org.cru.godtools.shared.renderer.state.State
 import org.cru.godtools.shared.tool.parser.model.Tabs
 
-private val TAB_SELECTED_COLOR = Color.DarkGray
-private val TAB_UNSELECTED_COLOR = Color.LightGray
-private val TAB_SELECTED_CONTENT_COLOR = Color.White
-private val TAB_UNSELECTED_CONTENT_COLOR = Color.DarkGray
+private val CORNER_RADIUS = 10.dp
 
 @Composable
 internal fun RenderTabs(tabs: Tabs, state: State, modifier: Modifier = Modifier) {
     val selectedIndex = remember { mutableStateOf(0) }
 
+    var firstTab: Tabs.Tab? = tabs.tabs.firstOrNull()
+    val borderColor: Color = firstTab?.manifest?.primaryColor?.toComposeColor() ?: Color.Black
```

Comment:
you won't want to use `manifest` here, that may skip a `primaryColor` on the page containing the `Tabs`. you should be able to do `tabs.stylesParent.primaryColor`. I would base the border color on the tabs component, and not the first tab